MIGRATING_TO_V8.0.0.MD

```diff
@@ -0,0 +1,214 @@
+```js
+// Setting the custom fields of a profile with a camel cased custom key
+await sdk.profiles.update(profile.id, {
+    customFields: {
+        helloWorld: 'test',
+    },
+});
+
+// Before v8.0.0 the customFields of the profile would be stored as
+// { 
+//     hello_world: 'test' 
+// }
+
+// After v8.0.0 the customFields of the profile would be stored as
+// {
+//     helloWorld: 'test'
+// }
+```
+
+### Changes to response data
+For all [affected properties](###-affected-properties) with the response mentioned in v8.0.0 the response will be returned as it was provided.
+
+#### Camel Casing Custom Keys
+
+If you are providing custom keys in snake case before v8.0.0, you will now receive them as snake case in the responses as well.
+While before you would receive them as camel case.
+
+Before v8.0.0 some responses used to be camel cased:
+```js
+// Setting the general configuration with a snake cased custom key
+await sdk.configurations.general.update({ 
+  data: {
+      enable_this_feature: true,
+    }
+});
+
+// Retrieving the general configuration with the custom key
+const configuration = await sdk.configurations.general.find();
+
+
+// Before v8.0.0 the custom key would be accessed as
+configuration.data.enableThisFeature
+
+// After v8.0.0 the custom key would be accessed as
+configuration.data.enable_this_feature 
+```
+
+#### Dates
+
+If you are providing custom keys with timestamp in the name, they will now not return as Javascript date objects anymore.
+You will now receive them as what was provided in the request.
+Except if you provide a date object, you will receive an iso string.
+
+Before v8.0.0 some timestamps in responses were returned as Javascript date objects:
+
+```js
+// Setting the general configuration with a snake cased custom key
+await sdk.configurations.general.update({ 
+  data: {
+      creationTimestamp: 0,
+      expiryTimestamp: new Date(2012,0,1),
+      updateTimestamp: '2010-02-05T00:00:00.000Z',
+    }
+});
+
+// Retrieving the general configuration with the custom key
+const configuration = await sdk.configurations.general.find();
+
+
+// Before v8.0.0 the data object in the response would be
+// {
+//     creationTimestamp: new Date(0), 
+//     expiryTimestamp: new Date(2012,0,1), 
+//     updateTimestamp: new Date('2010-02-05T00:00:00.000Z')
+// }
+
+// After v8.0.0 the data object in the response would be
+// {
+//     creationTimestamp: 0,
+//     expiryTimestamp: '2012-01-01T00:00:00.000Z',
+//     updateTimestamp: '2010-02-05T00:00:00.000Z',
+// }
+```
+
+#### Records Affected
+
+If you were providing custom keys with the name `records_affected` or `recordsAffected` they will now return as they are provided.
+While before they would return as `affectedRecords`.
+
+### Reverting to request normalization of custom keys for a single operation
+You may enable normalization of custom keys by setting the option `sdk.service.operation({ normalizeCustomData: true })`
```

Comment:
(minor) I like how you talk about "can be reverted" in the first sentence in the section below. Or maybe a "reenable" instead of just "enable" here.